bot.py

```diff
@@ -1697,6 +1695,18 @@ async def before_autoupdate(self):
             self.autoupdate.cancel()
             return
 
+    @tasks.loop(hours=1, reconnect=False)
+    async def log_expiry(self):
+        log_expire_after = self.config.get("log_expiration")
+        if log_expire_after == isodate.Duration():
+            return self.log_expiry.stop()
```

Comment:
What about just doing `return` here without stopping the loop?

That way if we set the `log_expiration` later, the loop will just continue to execute the next block without us needing to restart the bot.

Reply:
The reason why I stopped the loop instead of just returning is that the restart serves as a failsafe mechanism if someone accidentally sets a log expiration, and it just so happens to loop right before they can change it. An edge case scenario really.